spec/index.bs

```diff
@@ -3497,8 +3501,93 @@ The following validation rules apply:
             [=mipmap level=] |textureCopyView|.{{GPUTextureCopyView/mipLevel}}.
 </div>
 
+## Debug Markers ## {#command-encoder-debug-markers}
+
+Command encoders provide methods to apply debug labels to groups of commands or insert a single
+label into the command sequence. Debug groups can be nested to create a hierarchy of labeled
+commands. These labels MAY be passed to the native API backends for tooling, MAY be used by the user
+agent's internal tooling, or MAY be a no-op when such tooling is not available or applicable.
+
+Debug groups in a {{GPUCommandEncoder}} or {{GPUProgrammablePassEncoder}}
+must be well nested.
```

Comment:
This should also validate that the stack is empty on encoder finish / pass end.

Reply:
I left that one off initially because the finish/end methods weren't documented at all yet, and this didn't strike me as the right PR to add them? They were going to be the next thing I tackled, but I'm happy to move that to this change if you'd like.

Reply:
Makes sense. Could you add an `Issue:` just in case so we don't forget?

Reply:
Done.